tests/utils/misc/misc.go

```diff
@@ -47,3 +48,18 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+// LogTestStart - Print a start log with given test name and
+// current time stamp
+func LogTestStart(c *C, testGroup string, testName string) {
```

Comment:
I guess it might be possible to get the test group and test case name using reflection. That way it will be much easier to use these tracing functions. We can investigate this later.